src/components/image/__snapshots__/index.test.tsx.snap

```diff
@@ -1,6 +1,12 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Image component Image functional tests When getUrl returns null, with no errorComponent, should throw an error 1`] = `"URL not found for given property"`;
+exports[`Image component Image functional tests When getUrl returns null, with an errorComponent, should render the error 1`] = `
+<DocumentFragment>
+  <span>
+    Error: No value found for property.
```

Comment:
I have literally no context here, but seeing such an error message without telling me **_what_** property we found no value for seems dubious...?

Reply:
The test is just testing that _an_ error message is rendered, so the text itself isn't all that important - just that whatever text the test specifies is rendered.